client/src/pages/chart.jsx

```diff
@@ -17,24 +17,30 @@ import { getPortfolioItems } from '../services/portfolioService';
 import Portfolio from '../components/Portfolio';
 import StockSelector from '../components/StockSelector';
 import '../styles/app.css';
-class ChartComponent extends Component {
+import { timeParse } from "d3-time-format";
 
+const parseDate = timeParse("%s");
+
+class ChartComponent extends Chart {
```

Comment:
Why are we extending the `Chart` component here?  For what reason?

Reply:
Remnant of old attempt to get this working, changed it back to extend Component.